cypress/integration/chi-js/enterprise-global-nav.spec.js

```diff
@@ -0,0 +1,395 @@
+describe('enterprise-nav-menu', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/enterprise-global-nav.html');
+  });
+
+  describe('check that click on 1st level opens 2nd-level menu drawer', () => {
+    [/^NoChanges$/, /^Changes$/, /^DeepLink$/].forEach((menuText) => {
+      it('check "' + menuText + '" opens 2nd-level menu drawer', () => {
+
+        cy.get('.enterprise-nav-menu').within(() => {
+          cy.get('.m-sidenav__list li')
+            .contains(menuText)
+            .as('firstLevelElement')
+          ;
+
+          cy.get('@firstLevelElement')
+            .should('have.attr', 'href')
+            .then(href => {
+              cy.get('' + href)
+                .as('drawer')
+              ;
+
+              cy.get('@firstLevelElement')
+                .click()
+                .wait(550)
+              ;
+
+              cy.get(href + '.-active')
+                .should('be.visible')
+              ;
+            });
+        });
+      });
+    });
+  });
+
+  describe('check that click on "X" in menu drawer closes drawer', () => {
+    [/^NoChanges$/, /^Changes$/, /^DeepLink$/].forEach((menuText) => {
+      it('check "' + menuText + '" drawer closes on "X"', () => {
+
```

Comment:
Remove one of these empty lines. Please consider use [prettier](https://prettier.io) for prettify the code. It will remove duplicate empty lines automatically.